apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -187,7 +187,7 @@ export async function regenerateQuestion(
 }> {
   const serverJob = await createServerJob({
     courseId,
-    type: 'llm_question_regen',
+    type: 'ai_question_regen',
```

Comment:
No need for abbreviated names, this isn't C 😉 If the other job is called `ai_question_generate`, this can be called `ai_question_regenerate`.

Reply:
changed to long-form in a133e9b738f1eb22f09f6fa83561847be451b8eb